docs/api-docs/apps/guide/apps-04-devloper-portal.md

```diff
@@ -1,87 +1,77 @@
 
+## Edit technical details
 
+In the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps), click the **Edit App** pencil icon to the right of the app you want to edit. At the top of the dialog that opens, click the **Technical** tab to edit enabled features, callback URLs, and OAuth scopes. To save your change, click the **Update & Close** button on the dialog's lower right. After saving, edits are effective immediately for all app users.
 
-## Viewing credentials
+![Technical Details](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/devtools-technical.png "Technical Details")
 
-Click **View Client ID** to view an app's API credentials.
+![OAuth Scopes](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/devtool-oauth-scopes.png "OAuth Scopes")
 
-![Developer Portal](https://storage.googleapis.com/bigcommerce-production-dev-center/images/apps-04-developer-portal-01.png "Developer Portal")
+## Enable multi-storefront functionality
 
-[Learn how to use app credentials in the app OAuth flow](/api-docs/apps/guide/auth).
+If your app supports stores that sell through multiple storefronts or sales channels, add that information to the app's profile.
 
-<!-- theme: info -->
-> #### Note
-> * [OAuth Client ID is no longer required to authenticate requests to api.bigcommerce.com](/changelog#posts/o-auth-client-id-is-no-longer-required-for-requests-to-api-bigcommerce-com).
+In the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps), click the **Edit App** pencil icon to the right of the app you want to edit. At the top of the dialog that opens, click the **App Supported Features** tab. Select either **Single Storefront** or **Multi Storefront**, then save your change by clicking the **Update & Close** button on the dialog's lower right.
 
+To learn more about modifying your app to support multi-storefront, see [Multi-Storefront App Compatibility and Optimization](/api-docs/apps/multi-storefront)
 
+![MSF Toggle](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/devtools-msf.png "Toggle MSF on the App Supported Features tab")
 
-## Submitting apps
+## Submit an app for approval
 
-Submit apps for [Apps Marketplace](https://www.bigcommerce.com/apps) approval by clicking **Submit**.
+Submit apps for [Apps Marketplace](https://www.bigcommerce.com/apps) approval by navigating to the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps), then finding the app you want approved and clicking the **Submit** checkmark icon to the right of its listing. If you've left any required fields incomplete, the Dev Portal will prompt you to complete the app's profile. [Learn more about completing app registration fields and submitting apps for approval](/api-docs/apps/guide/publishing).

```

Comment:
```suggestion
Submit apps for [Apps Marketplace](https://www.bigcommerce.com/apps) approval by navigating to the Dev Portal's [My Apps dashboard](https://devtools.bigcommerce.com/my/apps), then finding the app you want to be approved and clicking the **Submit** checkmark icon to the right of its listing. The Dev Portal will prompt you to complete the app's profile if you've left any required fields incomplete. [Learn more about completing app registration fields and submitting apps for approval](/api-docs/apps/guide/publishing).

```

Reply:
worked with this and ultimately declined.